### ISSUE TITLE
Name Change Pending Logo Submission

### ISSUE THREAD

**Issue by marcaaron** [marcaaron/ww-calc-2.0#17]
Jen has suggested we change the name to **I-1600 Healthcare Estimator**.
Harry offered **I-1600 Healthcare Cost Estimator** as an alternative.
Previous discussion between Pamela and myself settled on the current name so I'm going to flag this here, but leave it as not actionable until we can agree on the change.

If updated we'll need someone to change it anywhere it appears on site e.g. the title, header, any twitter/fb meta tags etc.

Incorporating the logo (still being worked on) into the overall design will also impact how we display the header, display name, etc.

**Comment by couchpilotnye** [comment]
We're good with Healthcare Savings Estimator. 
I'm attaching some files of our new logo we're using on social media. Would you rather work directly with the graphic artist?
![logo_3-28-18](https://user-images.githubusercontent.com/37560660/38450479-28c85eaa-39d3-11e8-90fc-f56664947e57.png)

![yes1600_twitter_coverpic_wide](https://user-images.githubusercontent.com/37560660/38450468-fa1ced0a-39d2-11e8-9323-bd67b65498dc.png)

**Comment by marcaaron** [comment]
Yes, let's get together with the graphic artist on this stuff ! 
Maybe they'll have some input on how to incorporate things into the calculator design?

**Comment by marcaaron** [comment]
Ok converted the logo to svg and incorporated ... here's what things are looking like at the moment. Maybe someone has input on the best way to style the header?

![screen shot 149](https://user-images.githubusercontent.com/32969087/38517290-51a5abf8-3bee-11e8-9fb6-a3c72b1ea7af.png)
